src/main/java/com/urbancode/jenkins/plugins/ucdeploy/VersionHelper.java

```diff
@@ -295,6 +295,8 @@ private void setComponentVersionProperties(
                 propSheetDef = compClient.getComponentVersionPropSheetDef(component);
                 propSheetDefId = (String) propSheetDef.get("id");
                 propSheetDefPath = (String) propSheetDef.get("path");
+                propSheetDefPath = propSheetDefPath.replaceAll("/", "&");
```

Comment:
Hi Nick,

I don't have access to the PMR. Another team at Fidelity opened it. Where can I get your test code? Can you send it via email and I'll run it?

Thanks,
Mitch Goroff
mitchell.goroff@fmr.com